getting-started/bookshelf/bookshelf.go

```diff
@@ -38,8 +39,8 @@ type BookDatabase interface {
 	// ListBooks returns a list of books, ordered by title.
 	ListBooks(context.Context) ([]*Book, error)
 
-	// GetBook retrieves a book by its ID.
-	GetBook(ctx context.Context, id string) (*Book, error)
+	// Book retrieves a book by its ID.
+	Book(ctx context.Context, id string) (*Book, error)
```

Comment:
I prefer GetBook, matches other {Verb}Book methods in this interface.

Don't feel strongly though.